src/widgetastic/browser.py

```diff
@@ -413,24 +411,10 @@ def is_displayed(self, locator, *args, **kwargs):
             A :py:class:`bool`
         """
         kwargs['check_visibility'] = False
-        retry = True
-        tries = 10
-        while retry:
-            retry = False
-            try:
-                return self.move_to_element(locator, *args, **kwargs).is_displayed()
-            except (NoSuchElementException, MoveTargetOutOfBoundsException):
-                return False
-            except StaleElementReferenceException:
-                if isinstance(locator, WebElement) or tries <= 0:
-                    # We cannot fix this one.
-                    raise
-                retry = True
-                tries -= 1
-                time.sleep(0.1)
-
-        # Just in case
-        return False
+        try:
```

Comment:
This method isn't decorated with `@retry_stale_element`, how is it handled in this case?

Reply:
it is - the diff is just bad expanding puts it together

Reply:
well,I see it now, but I think a commit was added, either way, as long as its there I'm happy :)

Reply:
I removed decorator while I was fighting against PhantomJS issues. I've just restored it.
Besides that I decorate only methods which have been noticed causing issues.

Reply:
yep, you're right. I'm testing wt.core right now locally. So, I restore deleted parts according to test results.